packages/react/src/themes/teams-high-contrast/components/Button/buttonStyles.ts

```diff
@@ -1,43 +0,0 @@
-import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '../../../types'
-import { ButtonProps } from '../../../../components/Button/Button'
-import { ButtonVariables } from '../../../teams/components/Button/buttonVariables'
-import { ButtonHighContrastVariables } from './buttonVariables'
-
-const buttonStyles: ComponentSlotStylesPrepared<
-  ButtonProps,
-  ButtonVariables & ButtonHighContrastVariables
-> = {
-  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
```

Comment:
Are these files remove by intention?

Reply:
They were at one point but it looks like we should just revert, so I'm reverting them.